config/examples/Daycom/3DP-100/Configuration.h

```diff
@@ -737,7 +737,7 @@
  * Override with M92
  *                                      X, Y, Z, E0 [, E1[, E2...]]
  */
-#define DEFAULT_AXIS_STEPS_PER_UNIT   { 80, 80, 800, 94.3 }
+#define DEFAULT_AXIS_STEPS_PER_UNIT   { 80, 80, 4000, 500 }
```

Comment:
Thanks for merging the PR. :+1:

@thinkyhead Why was this line actually removed? My understanding ist that especially the steps per unit are model-specific and should be included in an example configuration. `{ 80, 80, 800, 94.3 }` are the values provided by the manufacturer and known to work.

Reply:
The configs were not up to date, so I had to update them, and as I do this several times a day in a rush mistakes are bound to happen. To prevent this in future make sure the configs are as up to date as possible so that no fixups will be required by the overworked maintainers.

Reply:
Ah sorry, I just now saw the default configuration in this repository on which I should have based my changes... Thanks for the clarification!